src/components/chip-set/examples/chip-set-input.tsx

```diff
@@ -49,10 +49,11 @@ export class ChipSetInputExample {
             this.createChip('Fish'),
         ];
 
-        this.value[0].iconColor = 'var(--lime-red)';
-        this.value[1].iconColor = 'var(--lime-orange)';
-        this.value[2].iconColor = 'var(--lime-green)';
-        this.value[3].iconColor = 'var(--lime-blue)';
+        this.value[0].iconFillColor = 'var(--lime-red)';
+        this.value[1].iconBackgroundColor = 'var(--lime-orange)';
+        this.value[2].iconBackgroundColor = 'var(--lime-green)';
+        this.value[3].iconFillColor = 'var(--lime-yellow)';
+        this.value[3].iconBackgroundColor = 'var(--lime-dark-blue)';
```

Comment:
The example looks pretty bad with these styles, I think. Could we get it to look a little nicer? I think there needs to be a good balance between showing how to do things technically, and setting a good example for how things ought to look.

Reply:
🤔 do you mean using CSS instead of writing things like this in the code?

Reply:
No no, I just meant purely visually, I think this looks a bit all over the place:

![image](https://user-images.githubusercontent.com/154725/110001341-948cb780-7d14-11eb-89c1-2c9d71593dc1.png)

Reply:
maybe we can reduce it to 1 example for each case? 
- only icon color
- only background color
- both icon and background color

Otherwise I don't have any problems with the example itself neither :)

Reply:
Also ⚡'ed by moving all colours to a separate example.